AdminCore.WebApp/Models/DataTransform/EventDataTransformModel.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Collections.Generic;
+using AdminCore.Constants.Enums;
+using AdminCore.DataETL.Attributes;
+using AdminCore.DataETL.TypeConverters;
+
+namespace AdminCore.WebApi.Models.DataTransform
+{
+    public class EventDataTransformModel
+    {
+        [ExportableRecordField(Name = "Event Id", ColumnPosition = 1)]
```

Comment:
should these be constants? maybe shared constants? thinking out loud.

Reply:
Which part specifically? The 'Name' parameter?

Reply:
yep, as discussed offline

Reply:
Decided to keep these attributes the way they are and not store 'Name' in as constants.